### PR TITLE
news_migration_1

### PR DESCRIPTION
**Pull request by kennethgillen**
News migration contribution.

### CODE REVIEW

**Comment by kennethgillen** [comment]
Note, I saw a couple of spelling issues when I had a peek across all the `.md` in the posts directory. Noted on Trello, too.

LS27172:_posts kenny$ grep -r clarifed 
2015-09-07-bio-formats-5-1-4.md: clarifed JVM versions currently supported

LS27172:_posts kenny$ grep -r Formata *
2016-02-18-letters-of-support-for-bio-formats-development-and-extensions.md:...for future work on Bio-Formata and...

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Release of Bio-Formats 5.1.4
```
Comment (mtbc) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: 5.1.8

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- clarified description of Qu for MATLAB (thanks to Carnë Draug)
+- added installation instructions for Octave (thanks to Carnë Draug)
+
+C++:
+
+- Bugfixes to the OME-TIFF writer to correct use of the metadata store with multiple series
+- Ensure file and writer state consistency upon close failure
+
+The software is available at:
+http://downloads.openmicroscopy.org/bio-formats/5.1.8
```
Comment (mtbc) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: turn into archived downloads link, see https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The C++ superbuilds and platform builds will be released shortly.
+
+Any problems or comments, please use the OME Forums or mailing lists:
+
+http://www.openmicroscopy.org/site/community
```
Comment (mtbc) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: follow https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-ec2b783a226c3c88e10591c03511bf91R45

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Regards,
```
Comment (mtbc) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: remove signoff

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Release of Bio-Formats 5.1.4
+intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats 5.1.8
+---
+Dear All,
```
Comment (mtbc) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: remove this bit

File: _posts/2016-02-18-image-data-repository-idr-project-update.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: Image Data Repository (IDR) project update
+intro-blurb: A webinar by Prof Jason Swedlow
+---
+A webinar in which Prof Jason Swedlow presents an update on the development of a new Euro-BioImaging Image Data Repository (IDR) by University of Dundee, University of Cambridge and EMBL-EBI, and its interface with ELIXIR is now available at https://www.elixir-europe.org/documents/elixir-webinar-update-euro-bioimaging-image-data-repository-idr-project-and-its-interface
```
Comment (mtbc) [inline _posts/2016-02-18-image-data-repository-idr-project-update.md]: maybe wrap link with "available for download" text or somesuch?

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Release of Bio-Formats-C++ 5.1.8
+intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats-C++ 5.1.8
+---
+Dear All,
```
Comment (mtbc) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: remove this

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+-  py-sphinx 1.2.3
+-  tiff 4.0.6
+-  xerces-c 3.1.2
+-  zlib 1.2.8
+
+The software release is available at: http://downloads.openmicroscopy.org/bio-formats-cpp/5.1.8
+
+For any problems or comments, please use the OME forums or mailing lists: http://www.openmicroscopy.org/site/community
+
+Roger Leigh
```
Comment (mtbc) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: remove signoff

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+-  py-pygments 2.0.2
+-  py-setuptools 18.3.2
+-  py-sphinx 1.2.3
+-  tiff 4.0.6
+-  xerces-c 3.1.2
+-  zlib 1.2.8
+
+The software release is available at: http://downloads.openmicroscopy.org/bio-formats-cpp/5.1.8
+
+For any problems or comments, please use the OME forums or mailing lists: http://www.openmicroscopy.org/site/community
```
Comment (mtbc) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: change as for the BF release

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+-  py-jinja2 2.7.3
+-  py-markupsafe 0.23
+-  py-pygments 2.0.2
+-  py-setuptools 18.3.2
+-  py-sphinx 1.2.3
+-  tiff 4.0.6
+-  xerces-c 3.1.2
+-  zlib 1.2.8
+
+The software release is available at: http://downloads.openmicroscopy.org/bio-formats-cpp/5.1.8
```
Comment (mtbc) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: change as for the BF release (archived downloads)

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Release of Bio-Formats-C++ 5.1.8
+intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats-C++ 5.1.8
+---
+Dear All,
+
+Today we have released the binary builds of Bio-Formats-C++ 5.1.8, together with the ome-cmake-superbuild package used for building Bio-Formats-C++ and its dependencies on Windows and other platforms.  With our current C++ development efforts being devoted to the upcoming 5.2.0 release, this release contains no new features, and is restricted to security updates in our third-party dependencies.  Future 5.1 releases will also be restricted to serious bugfixes and security updates.
+
+See http://downloads.openmicroscopy.org/bio-formats-cpp/5.1.8/ for links to the source releases, documentation and API reference, and binary builds for a number of platforms including Windows, Linux and MacOS X.
```
Comment (mtbc) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: not sure how best to hide this link behind text, @hflynn may want to consider next week

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md (hunk 5 above)
Comment (kennethgillen) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: ✔️

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md (hunk 4 above)
Comment (kennethgillen) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: ✔️

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md (hunk 3 above)
Comment (kennethgillen) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: ✔️

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md (hunk 2 above)
Comment (kennethgillen) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: ✔️

File: _posts/2016-02-15-release-of-bio-formats-5.1.8.md (hunk 1 above)
Comment (kennethgillen) [inline _posts/2016-02-15-release-of-bio-formats-5.1.8.md]: ✔️

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md (hunk 7 above)
Comment (kennethgillen) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: ✔️

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md (hunk 8 above)
Comment (kennethgillen) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: ✔️

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md (hunk 9 above)
Comment (kennethgillen) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: ✔️

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md (hunk 10 above)
Comment (kennethgillen) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: ✔️

File: _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md (hunk 11 above)
Comment (kennethgillen) [inline _posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md]: trying 'archived downloads' as other similar links

**Pull request merged**